<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontPage</class>
 <widget class="QMainWindow" name="FrontPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="MainMenu">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>281</width>
          <height>524</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/images/screen1.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>281</width>
          <height>524</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/images/screen2.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>10</y>
          <width>281</width>
          <height>524</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/images/screen3.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>10</y>
          <width>281</width>
          <height>524</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/img/images/screen4.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="LearnMore"/>
      <widget class="QWidget" name="ContactUs"/>
      <widget class="QWidget" name="CustomerTestimony"/>
      <widget class="QWidget" name="AdminLogin">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>140</y>
          <width>391</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>Admin Sign-in</string>
        </property>
        <widget class="QLineEdit" name="username">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>291</width>
           <height>24</height>
          </rect>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Enter Username...</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="password">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>291</width>
           <height>24</height>
          </rect>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Enter Password...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loginButton">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>130</y>
           <width>80</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="goBackButtonAdminLogin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-- Go Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="AdminCommands">
       <widget class="QPushButton" name="goBackButtonAdminCommands">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-- Go Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadDatabaseButton">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>500</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Database</string>
        </property>
       </widget>
       <widget class="QToolButton" name="helpButtonAdminCommands">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>500</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
       <widget class="QListView" name="customerListView">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>140</y>
          <width>981</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addCustomerButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>500</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Customer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdminCommandsSaveChanges">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>500</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Changes</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AdminCommandsDiscardChanges">
        <property name="geometry">
         <rect>
          <x>990</x>
          <y>500</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Discard Changes</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteCustomerButton">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>500</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete Customer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchBarAdminCommands">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>93</y>
          <width>981</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Type Here to Search...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendPamphletButton">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>500</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Send Pamphlet</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>231</width>
          <height>381</height>
         </rect>
        </property>
        <property name="title">
         <string>Edit Changes</string>
        </property>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Company Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="editWebsiteName">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>250</y>
           <width>211</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>230</y>
           <width>201</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Website Name</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Interest Level</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="editCompanyName">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>211</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="editCityStateZip">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>211</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Address 1</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Address 2</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>330</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Key Customer</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="editInterestLevel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>300</y>
           <width>211</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="editKeyCustomer">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>350</y>
           <width>211</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="editStreetName">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>211</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="displayCompanyID">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>211</width>
           <height>24</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Company ID</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>FrontPage</receiver>
   <slot>displayLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>1145</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayLogin()</slot>
 </slots>
</ui>
